src/index.js

```diff
@@ -33,16 +34,19 @@ registerPlugin( 'wc-calypso-bridge', {
 	),
 } );
 
-
 if ( !! window.wcCalypsoBridge.isEcommercePlanTrial ) {
-	// Unregister 'wc-admin-onboarding-task-payments'' task from WooCommerce Core
-	// Otherwise we'll have both the original payments and trial payments rendered.
+	// Unregister task fills from WooCommerce Core
+	// Otherwise we'll have both the original and new fills rendered.
+	const oldTaskNames = [
+		'wc-admin-onboarding-task-payments',
+		'wc-admin-onboarding-task-tax',
+	];
 	addAction(
 		'plugins.pluginRegistered',
 		'wc-calypso-bridge',
 		function ( _settings, name ) {
-			if ( name === 'wc-admin-onboarding-task-payments' ) {
-				unregisterPlugin( 'wc-admin-onboarding-task-payments' );
+			if ( oldTaskNames.indexOf( name ) !== -1 ) {
```

Comment:
I think we can use `includes` here, though they are very similar. 😄 

```suggestion
			if ( oldTaskNames.includes( name ) ) {
```